detections/endpoint/windows_suspicious_driver_loaded_path.yml

```diff
@@ -15,12 +15,12 @@ description: The following analytic detects the loading of drivers from
   leading to privilege escalation, persistence, or further system compromise.
 data_source:
 - Sysmon EventID 6
-search: '`sysmon` EventCode=6 ImageLoaded = "*.sys" NOT (ImageLoaded IN("*\\WINDOWS\\inf","*\\WINDOWS\\System32\\drivers\\*",
-  "*\\WINDOWS\\System32\\DriverStore\\FileRepository\\*","*:\Windows\\WinSxS\\*","*\\ProgramData\\Microsoft\\Windows
-  Defender\\Definition Updates\\*")) |  stats  min(_time) as firstTime max(_time)
-  as lastTime count by ImageLoaded dest dvc process_hash process_path signature signature_id
-  user_id vendor_product | `security_content_ctime(firstTime)` | `security_content_ctime(lastTime)`
-  | `windows_suspicious_driver_loaded_path_filter`'
+search: '`sysmon` EventCode=6 
+   NOT (ImageLoaded IN("*\\WINDOWS\\inf","*\\WINDOWS\\System32\\drivers\\*", "*\\WINDOWS\\System32\\DriverStore\\FileRepository\\*","*:\Windows\\WinSxS\\*","*\\ProgramData\\Microsoft\\Windows Defender\\Definition Updates\\*")) 
```

Comment:
This needs some fixing

- `\\WINDOWS\\inf` is missing a wildcard
- Sometimes `WINDOWS` is reported as `SystemRoot` so i suggest you add that option for all.
- `ProgramData`, `Program Files` and `Program Files (x86)` are also legit places that could load a driver.

Overall I suggest you make the list a whitelist instead of a blacklist or else we would have to reduce this to hunting rule